src/modules/photos/componentes/Contrast.vue

```diff
@@ -0,0 +1,66 @@
+<template>
+  <button @click="startBrilho">Brilho e Contraste</button>
+
+  <div v-if="showBrilho">
+    <label for="brightness">Brilho:</label>
+    <input
+      type="range"
+      id="brightness"
+      v-model="brightness"
+      min="-100"
+      max="100"
+      step="1"
+    />
+    <label for="contrast">Contraste:</label>
+    <input
+      type="range"
+      id="contrast"
+      v-model="contrast"
+      min="-100"
+      max="100"
+      step="1"
+    />
+    <button @click="applyFilters">Aplicar Filtros</button>
+    <button @click="resetFilters">Redefinir Filtros</button>
+    <button @click="showBrilho = false">Cancelar</button>
+  </div>
+</template>
+
+<script>
+export default {
+  data() {
+    return {
+      showBrilho: false,
+    };
+  },
+  methods: {
+    startBrilho() {
+      this.showBrilho = true;
+    },
+    applyFilters() {
+      if (this.fabricImage) {
```

Comment:
não consegui enxergar de onde vem esse this.fabricImage. É injetado pela biblioteca? 

Além disso, seria legal verificar se ele não está definido e já retornar, ao invés de manter muitas coisas dentro de um if.
```suggestion
      if (!this.fabricImage) {
```